nncf/quantization/advanced_parameters.py

```diff
@@ -154,7 +169,6 @@ class AdvancedQuantizationParameters:
     inplace_statistics: bool = True
     disable_channel_alignment: bool = True
     disable_bias_correction: bool = False
-    smooth_quant_alpha: float = 0.95
```

Comment:
This is not backward compatible changes. I suggest to depreciate  smooth_quant_alpha in this release and remove in the next release.

Reply:
Do you have any ideas on how to keep new and old arguments together since they are called almost the same?

Reply:
I hope, the deprecation warning and mapping smooth_quant_alph to AdvancedSmoothQuantParameters. alpha will be enough.

Reply:
Added warnings.